packages/calypso-products/src/translations.tsx

```diff
@@ -261,6 +261,40 @@ export const getJetpackProductsTaglines = (): Record<
 	};
 };
 
+export const getJetpackProductDisclaimers = ( link: string ): Record< string, TranslateResult > => {
+	const getLink = () => {
+		if ( link[ 0 ] === '#' ) {
+			return <a href={ link }></a>;
+		}
+
+		return <a href={ link } target="_blank" rel="noreferrer"></a>;
+	};
+
+	const backupT1Disclaimer = translate(
+		'* Subject to your usage and storage limit. {{link}}Learn more{{/link}}.',
+		{
+			components: {
+				link: getLink(),
+			},
+		}
+	);
+	const backupT2Disclaimer = translate(
+		'* Subject to your usage and storage limit. {{link}}Learn more{{/link}}.',
+		{
+			components: {
+				link: getLink(),
+			},
+		}
+	);
+
+	return {
+		[ PRODUCT_JETPACK_BACKUP_T1_YEARLY ]: backupT1Disclaimer,
+		[ PRODUCT_JETPACK_BACKUP_T1_MONTHLY ]: backupT1Disclaimer,
+		[ PRODUCT_JETPACK_BACKUP_T2_YEARLY ]: backupT2Disclaimer,
+		[ PRODUCT_JETPACK_BACKUP_T2_MONTHLY ]: backupT2Disclaimer,
+	};
```

Comment:
Nice catch! So strange that this only happens when a site owns Jetpack Security. On the same page on a site that has Jetpack Complete, it shows up just fine. I'll take a look at this

Reply:
Oh! Hmm, yes that _is_ strange. 🤔 I'm not sure offhand what would be causing that. 🤔

Reply:
The problem those cards technically have the "plan" type of item instead of the "product" because it wants to upgrade the JP Security plan. So I added it to the plan builder but now it's showing up on the pricing page on the security plan 😭. Will fix this next hahaha

Reply:
Oh MAN was that a rabbithole haha, turns out in `client/my-sites/plans/jetpack-plans/storage-tier-upgrade/hooks/use-get-storage-upgrade-product.ts`, the "features" are bing overridden manually after the `client/my-sites/jetpack-plans/slug-to-selector-product.ts` function (that is used for all product cards) runs. 

Therefore, the disclaimer that I was setting in `slug-to-selector-products.ts` wasn't using the proper features to determine whether or not to display. So the only solution I could think of was to add the disclaimer in the `use-get-storage-upgrade-product` function as well so it uses the correct feature set to determine whether or not to show. 

Anyway, long story short, problem should be fixed!

Reply:
Oh wow, tricky.. Nice work figuring that out! 😀